src/shipping/handlers.ts

```diff
@@ -43,14 +46,32 @@ export const createChannelsChangeHandler = (
   triggerChange();
 };
 
+const getaddPostalCodeRules = (
```

Comment:
This function name isn't clear on what it does, perhaps `getAddedPostalCodeRules` makes more sense?